charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -17,7 +17,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 11.0.2
+version: 11.0.3
```

Comment:
```suggestion
version: 11.0.4
```

Reply:
We need to bump the version again. If you could accept the suggestion above please?